src/cryptography/hazmat/bindings/openssl/x509_vfy.py

```diff
@@ -29,9 +29,22 @@
 typedef ... Cryptography_STACK_OF_ASN1_OBJECT;
 
 typedef ... X509_STORE;
-typedef ... X509_STORE_CTX;
 typedef ... X509_VERIFY_PARAM;
 
+typedef struct x509_store_ctx_st {
```

Comment:
I think usually we don't include the name of the struct here? So just `typedef struct {` should work